### ISSUE TITLE
Tutor formatting breaks out of the box in 0.4.0+

### ISSUE THREAD

**Issue by killthekitten** [neovim/neovim#11401]
- `nvim --version`:

This is the earliest I could get it to reproduce, but also reproduces in all the future versions, up to `HEAD-e3b08a0`.

```
NVIM v0.4.0
Build type: Release
LuaJIT 2.0.5
Compilation: /Applications/Xcode-10.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -O2 -DNDEBUG -DMIN_LOG_LEVEL=3 -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wmissing-prototypes -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fdiagnostics-color=auto -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -I/Users/travis/build/neovim/bot-ci/build/neovim/build/config -I/Users/travis/build/neovim/bot-ci/build/neovim/src -I/Users/travis/build/neovim/bot-ci/build/neovim/.deps/usr/include -I/usr/local/opt/gettext/include -I/usr/include -I/Users/travis/build/neovim/bot-ci/build/neovim/build/src/nvim/auto -I/Users/travis/build/neovim/bot-ci/build/neovim/build/include
Compiled by travis@Traviss-Mac-6.local

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/share/nvim"

Run :checkhealth for more info
```

- Operating system/version: macOS Catalina 10.15.1
- Terminal name/version: iTerm2 3.3.6
- `$TERM`: xterm-256color

### Steps to reproduce using `nvim -u NORC`

```
nvim -u NORC
:Tutor
:73
```

### Actual behaviour

Formatting breaks after (supposedly) line 73 throughout the rest of the file:

<img width="937" alt="Screenshot 2019-11-16 at 12 13 48" src="https://user-images.githubusercontent.com/1201569/68992442-d36ed380-086b-11ea-9b76-dc7331b6a902.png">

### Expected behaviour

0.3.8 and below:

<img width="937" alt="image" src="https://user-images.githubusercontent.com/1201569/68992532-f2ba3080-086c-11ea-9aa3-0a3bdb253acc.png">

Might require @fmoralesc's attention.

**Comment by jarenado** [comment]
is this still an issue? I think I fixed it, I'm just having a bit of trouble building it locally.

**Comment by killthekitten** [comment]
@jarenado could you reference the commit / PR so I can check?

**Comment by jarenado** [comment]
@killthekitten I think justin closed this. should I still file a PR? I got the build running and made the changes to the tutor file.

**Comment by killthekitten** [comment]
@jarenado I had an impression that you fixed it somewhere else and it’s already been merged ;)

@justinmk did you close this by mistake?

**Comment by jarenado** [comment]
Sorry, that was a poorly worded comment. I'll file a PR.

**Comment by limjh16** [comment]
Hey, sorry for resurrecting an old issue, but I am still facing this problem when I run `nvim -u NORC` and then `:Tutor`. I'm using iTerm2 3.3.9, and `$TERM` shows `xterm-256color`. Should I open a new issue?
I'm using v0.4.3 from Homebrew on MacOS Catalina 10.15.3
Here are some screenshots of the broken formatting
<img width="273" alt="Screenshot 2020-05-01 at 11 12 39 AM" src="https://user-images.githubusercontent.com/15180396/80778947-ad7c8780-8b9c-11ea-9574-8adebbda1c49.png">
<img width="257" alt="Screenshot 2020-05-01 at 11 13 21 AM" src="https://user-images.githubusercontent.com/15180396/80778972-c5eca200-8b9c-11ea-9443-f948509cffc5.png">
<img width="229" alt="Screenshot 2020-05-01 at 11 13 11 AM" src="https://user-images.githubusercontent.com/15180396/80778966-bf5e2a80-8b9c-11ea-9f90-53e46afa33f0.png">

**Comment by killthekitten** [comment]
@limjh16 the fix is most likely in the 0.5.0 build, could you try installing it?

**Comment by sergiopantoja** [comment]
Same here. Nvim 0.5.0, macOS Catalina 10.15.4, iTerm 3.3.9

```shell
$ brew install --HEAD neovim

$ nvim -v
NVIM v0.5.0-nightly
Build type: Release
LuaJIT 2.0.5
...

$ nvim
:Tutor
```
<img width="568" alt="Screen Shot 2020-05-01 at 6 33 59 PM" src="https://user-images.githubusercontent.com/2530982/80848860-5eedde00-8bda-11ea-8870-76456e07a97f.png">
